packages/fxa-auth-server/lib/senders/emails/partials/location/index.ts

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const location = `
+  <mj-include path="./lib/senders/emails/css/location/index.css" type="css" css-inline="inline" />
+  <mj-section>
+    <mj-column>
+      <mj-text css-class="text-body-grey">
+        <% if (locals.primaryEmail) { %>
+            <span><%- primaryEmail %></span><br />
+            <% } %> 
+        <% if (locals.device) { %>
+            <span><%- device %></span><br /> 
+            <% } %> 
+        <% if (locals.location) { %>
+            <span><%- location %></span><br /> 
+            <% } %> 
+        <% if (locals.ip) { %>
+            <span data-l10n-id="user-ip" data-l10n-args='<%= JSON.stringify({ip}) %>'>IP address: <%- ip %></span><br /> 
+            <% } %> 
+        <% if (locals.date) { %>
+            <span><%- date %></span><br /> 
+            <% } %> 
```

Comment:
2 nitpicks: we (at least in this codebase, this may not apply to other codebases) generally indent a closing bracket at the same level the opening bracket is at, and indent with 2 spaces 😀